core/src/main/java/org/infinispan/distribution/ch/impl/DefaultConsistentHashFactory.java

```diff
@@ -30,7 +30,7 @@
  * @author anistor@redhat.com
  * @since 5.2
  */
-public class DefaultConsistentHashFactory implements ConsistentHashFactory<DefaultConsistentHash> {
+public class DefaultConsistentHashFactory extends AbstractConsistentHashFactory<DefaultConsistentHash> {
```

Comment:
Seems I've screwed more when refactoring that - the builder should inherit from abstract builder